chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/pendingListing/reviewer/TestStandardReviewer.java

```diff
@@ -0,0 +1,56 @@
+package gov.healthit.chpl.validation.pendingListing.reviewer;
+
+import java.util.Iterator;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+
+import gov.healthit.chpl.dao.TestStandardDAO;
+import gov.healthit.chpl.dto.TestStandardDTO;
+import gov.healthit.chpl.dto.listing.pending.PendingCertificationResultDTO;
+import gov.healthit.chpl.dto.listing.pending.PendingCertificationResultTestStandardDTO;
+import gov.healthit.chpl.dto.listing.pending.PendingCertifiedProductDTO;
+import gov.healthit.chpl.util.ErrorMessageUtil;
+import gov.healthit.chpl.util.Util;
+
+@Component("pendingTestStandardReviewer")
+public class TestStandardReviewer implements Reviewer {
+    private ErrorMessageUtil msgUtil;
+    private TestStandardDAO testStandardDao;
+
+    @Autowired
+    public TestStandardReviewer(TestStandardDAO testStandardDao, ErrorMessageUtil msgUtil) {
+        this.msgUtil = msgUtil;
+        this.testStandardDao = testStandardDao;
+    }
+
+    @Override
+    public void review(PendingCertifiedProductDTO listing) {
+        for (PendingCertificationResultDTO cert : listing.getCertificationCriterion()) {
```

Comment:
This looks like the kind of thing that could be done with streams, though not positive